src/v2/view-builder/components/OdaOktaVerifyTerminalView.js

```diff
@@ -72,3 +85,76 @@ export default View.extend({
     });
   },
 });
+
+const AndroidAppLinkWithAccountOdaTerminalView =  BaseOdaOktaVerifyTerminalView.extend({
+  template: hbs`
+    <p class="explanation" data-se="subheader">
+    {{i18n code="enroll.oda.with.account.explanation" bundle="login"}}
+    </p>
+    <p class="subtitle">
+    {{i18n code="enroll.oda.with.account.subtitile1 " bundle="login"}}
+    </p>
+    <ul>
+      <li>{{i18n code="enroll.oda.with.account.step1" bundle="login"}}</li>
+      <li>{{i18n code="enroll.oda.with.account.step2" bundle="login"}}</li>
+      <li>{{i18n code="enroll.oda.with.account.step3" bundle="login"}}</li>
+    </ul>
+    <p class="subtitle">
+    {{i18n code="enroll.oda.with.account.subtitile2 " bundle="login"}}
+    </p>
+    <ol>
+      <li>{{i18n code="enroll.oda.with.account.step4" bundle="login"}}</li>
+      <li>{{i18n code="enroll.oda.with.account.step5" bundle="login" arguments="orgName"}}</li>
```

Comment:
looks like the mock is referring to orgUrl instead of name. We can use signInUrl for this purpose